src/press_scheduler.rs

```diff
@@ -0,0 +1,39 @@
+//! This module contains [`PressScheduler`] and its supporting methods and impls.
```

Comment:
This could be more useful: readers will already see this right below in the module docs.

Reply:
Should I add extra detail about how the press scheduler is supposed to be used here or on the `PressScheduler` type? Ideally I wouldn't put a doc comment on a file with such few items in the first place, but, deny(missing_docs) forced my hand :p

Reply:
Hmm. Module level docs please.